src/skeptic/lib.rs

```diff
@@ -784,3 +797,131 @@ fn test_sanitization_of_testnames() {
     assert_eq!(sanitize_test_name("my_long__fun___name___with____a_____lot______of_______spaces"), "my_long_fun_name_with_a_lot_of_spaces");
     assert_eq!(sanitize_test_name("Löwe 老虎 Léopard"), "l_we_l_opard");
 }
+
+#[test]
+fn test_extracting_tests_from_string_states_the_line_number_as_the_beginning_of_the_code_block() {
+    let lines = &[
+        "Rust code that should panic when running it.\n".to_owned(),
```

Comment:
these `to_owned` calls look noisy (this is an anti pattern that has been in the code for some time). I would suggest using raw string literals then iterating over the lines and mapping with to_owned and collecting into a vec.